examples/pytorch/grand/model.py

```diff
@@ -0,0 +1,104 @@
+import torch as th
+import torch.nn as nn
+import dgl.function as fn
+
+from layers import drop_node, MLP
+
+def GRANDConv(graph, feats, order):
```

Comment:
Any reason that you write two GRANDConv? I see one at the main.py file, and another one here.

Reply:
Sorry, I just forgot to remove these lines.